flutter/lib/resources/result_manager.dart

```diff
@@ -76,7 +76,7 @@ class _FullResult {
       'configuration': {
         'runtime': '',
       },
-      'score': score,
+      'throughput': throughput,
```

Comment:
Are we allowed to change format of `result.json`? I don't think we discussed it in the last meeting.

I remember that I was requested to compare formats of `result.json` generated by the old app and by the Flutter app.
We still have some strange things just to make `result.json` fully equivalent to the old app result file format. For example, saving numeric values in strings, and manually adding `\` to `/`, as if it is a special character.

Reply:
I created an issue to discuss it: https://github.com/mlcommons/mobile_app_open/issues/247

Reply:
We can probably ignore it for now, and revert this field name change later if `result.json` must have `score` field.